atlas/deployment.yaml

```diff
@@ -1,15 +0,0 @@
-apiVersion: atlas.mongodb.com/v1
```

Comment:
In order to minimize any future merge conflicts, can we keep the `atlas` directory intact? As long as our new manifest files aren't in this directory, it seems like it won't impact our new functionality right?